string id
int64 pos_x
int64 pos_y
int64 size_x
int64 size_y